chainer/functions/connection/convolution_2d.py

```diff
@@ -13,13 +13,7 @@
 
 if cuda.cudnn_enabled:
     cudnn = cuda.cudnn
```

Comment:
Maybe we don't have to define this global submodule? It seems to be used only once/twice.

Reply:
Thanks